cli/src/commands/token/create/dev.rs

```diff
@@ -99,80 +99,83 @@ async fn read_config(
 )> {
 	let config = commands::config::read_config(Vec::new(), Some(ns_name_id)).await?;
 
-	let Some(matchmaker) = &config.matchmaker else {
-		bail!("matchmaker not enabled");
-	};
-
-	let dev_hostname = matchmaker
-		.dev_hostname
-		.clone()
-		.unwrap_or_else(|| "127.0.0.1".to_string());
-
-	// Read lobby ports that we need to expose
-	let dev_ports = {
-		let mut lobby_ports =
-			HashMap::<String, &models::CloudVersionMatchmakerGameModeRuntimeDockerPort>::new();
-
-		// Register lobby ports from all game modes
-		if let Some(game_modes) = matchmaker.game_modes.as_ref() {
-			for game_mode in game_modes.values() {
-				if let Some(ports) = game_mode.docker.as_ref().and_then(|x| x.ports.as_ref()) {
-					for (label, port) in ports {
-						lobby_ports.insert(label.clone(), port);
+	if let Some(matchmaker) = &config.matchmaker {
```

Comment:
nit: thoughts on using implicit returns?

Reply:
Do you mean switching this to an `if let/else` and allowing both paths to return their result?